shell/platform/android/android_external_texture_gl.cc

```diff
@@ -20,7 +22,7 @@ AndroidExternalTextureGL::AndroidExternalTextureGL(
     const fml::jni::ScopedJavaGlobalRef<jobject>& surface_texture,
     std::shared_ptr<PlatformViewAndroidJNI> jni_facade)
     : Texture(id),
-      jni_facade_(jni_facade),
+      jni_facade_(std::move(jni_facade)),
```

Comment:
I'm not quite sure why sometimes it's okay with value arguments like this, but at least it's still a net positive because it's finding move locations.